content/next/index.md

```diff
@@ -53,19 +53,12 @@ BREAKING CHANGE: `extends` key in config file is now used for extending other co
 
 ### Commit message with `!` to draw attention to breaking change
 ```
-feat!: drop support for Node 6
+feat!: enforce authentication for all requests
 ```
 
 ### Commit message with scope and `!` to draw attention to breaking change
 ```
-refactor(runtime)!: drop support for Node 6
-```
-
-### Commit message with both `!` and BREAKING CHANGE footer
```

Comment:
I'd still keep both the example with both of them

Reply:
I dropped my first commit again and replaced the `refactor(runtime)!` through a `chore(runtime)!`. I think `chore` is the correct scope for dropping node support, as it is neither really a `feat` nor a `refactor` in my opinion.